src/components/userModal/UserModal.scss

```diff
@@ -81,4 +81,26 @@
     text-align: center;
     font-size: 9px;
     color: white;
-}
\ No newline at end of file
+}
+
+.yes-button {
+    margin-left: 8px;
+    margin-right: 8px;
+    padding: 8px 16px;
+    background-color: #4CAF50;
+    color: white;
+    border-radius: 4px;
+    border: none;
+    cursor: pointer;
+    font-size: 16px;
+  }
+
+.no-button {
+    padding: 8px 16px;
+    background-color: #f44336;
+    color: white;
+    border-radius: 4px;
+    border: none;
+    cursor: pointer;
+    font-size: 16px;
+}
```

Comment:
Could we wrap contents of this entire file in a top level component class to prevent it from conflicting with other scss. Something like

.TOPLEVELCLASS {
  .all_of_your_current_css_classes
}

You could refer App.js in case there is any confusion.